diff --git a/linux-3.4/drivers/video/sunxi/disp2/disp/dev_disp.c b/linux-3.4/drivers/video/sunxi/disp2/disp/dev_disp.c
index 7ad900b..e8b873d 100755
--- a/linux-3.4/drivers/video/sunxi/disp2/disp/dev_disp.c
+++ b/linux-3.4/drivers/video/sunxi/disp2/disp/dev_disp.c
@@ -1317,6 +1317,13 @@ long disp_ioctl(struct file *file, unsigned int cmd, unsigned long arg)
 		cptr = mgr->cptr;
 	}
 
+	if(cmd == DISP_CMD_VERSION) {
+		if(ubuffer[0] == 0x00010000) {
+			return 0;
+		}
+		return -1;
+	}
+
 	if(cmd < DISP_FB_REQUEST)	{
 		if(ubuffer[0] >= num_screens) {
 			__wrn("para err in disp_ioctl, cmd = 0x%x,screen id = %d\n", cmd, (int)ubuffer[0]);
diff --git a/linux-3.4/include/video/sunxi_display2.h b/linux-3.4/include/video/sunxi_display2.h
index 94231d3..3318522 100755
--- a/linux-3.4/include/video/sunxi_display2.h
+++ b/linux-3.4/include/video/sunxi_display2.h
@@ -425,7 +425,7 @@ typedef enum
 typedef enum tag_DISP_CMD
 {
 	//----disp global----
-	DISP_RESERVE0 = 0x00,
+	DISP_CMD_VERSION = 0x00,
 	DISP_RESERVE1 = 0x01,
 	DISP_SET_BKCOLOR = 0x03,
 	DISP_GET_BKCOLOR = 0x04,
