diff --git a/driver/src/devicedrv/mali/Kbuild b/driver/src/devicedrv/mali/Kbuild
index 6fcc361..3350272 100644
--- a/driver/src/devicedrv/mali/Kbuild
+++ b/driver/src/devicedrv/mali/Kbuild
@@ -172,6 +172,8 @@ endif
 # Use our defines when compiling
 ccflags-y += $(DEFINES) -I$(DRIVER_DIR) -I$(DRIVER_DIR)/include -I$(DRIVER_DIR)/common -I$(DRIVER_DIR)/linux -I$(DRIVER_DIR)/platform
 
+ccflags-y += -Wno-date-time
+
 # Source files which always are included in a build
 SRC = \
 	common/mali_kernel_core.c \
