diff --git a/driver/src/devicedrv/mali/linux/mali_kernel_sysfs.c b/driver/src/devicedrv/mali/linux/mali_kernel_sysfs.c
index 77ebc9d..bcdd3c4 100644
--- a/driver/src/devicedrv/mali/linux/mali_kernel_sysfs.c
+++ b/driver/src/devicedrv/mali/linux/mali_kernel_sysfs.c
@@ -123,7 +123,7 @@ static ssize_t gp_gpx_counter_srcx_write(struct file *filp, const char __user *u
 
 	buf[cnt] = 0;
 
-	ret = strict_strtol(buf, 10, &val);
+	ret = kstrtol(buf, 10, &val);
 	if (ret < 0)
 	{
 		return ret;
@@ -174,7 +174,7 @@ static ssize_t gp_all_counter_srcx_write(struct file *filp, const char __user *u
 
 	buf[cnt] = 0;
 
-	ret = strict_strtol(buf, 10, &val);
+	ret = kstrtol(buf, 10, &val);
 	if (ret < 0)
 	{
 		return ret;
@@ -327,7 +327,7 @@ static ssize_t pp_ppx_counter_srcx_write(struct file *filp, const char __user *u
 
 	buf[cnt] = 0;
 
-	ret = strict_strtol(buf, 10, &val);
+	ret = kstrtol(buf, 10, &val);
 	if (ret < 0)
 	{
 		return ret;
@@ -378,7 +378,7 @@ static ssize_t pp_all_counter_srcx_write(struct file *filp, const char __user *u
 
 	buf[cnt] = 0;
 
-	ret = strict_strtol(buf, 10, &val);
+	ret = kstrtol(buf, 10, &val);
 	if (ret < 0)
 	{
 		return ret;
@@ -536,7 +536,7 @@ static ssize_t l2_l2x_counter_srcx_write(struct file *filp, const char __user *u
 
 	buf[cnt] = 0;
 
-	ret = strict_strtol(buf, 10, &val);
+	ret = kstrtol(buf, 10, &val);
 	if (ret < 0)
 	{
 		return ret;
@@ -587,7 +587,7 @@ static ssize_t l2_all_counter_srcx_write(struct file *filp, const char __user *u
 
 	buf[cnt] = 0;
 
-	ret = strict_strtol(buf, 10, &val);
+	ret = kstrtol(buf, 10, &val);
 	if (ret < 0)
 	{
 		return ret;
@@ -814,7 +814,7 @@ static ssize_t profiling_record_write(struct file *filp, const char __user *ubuf
 
 	buf[cnt] = 0;
 
-	ret = strict_strtoul(buf, 10, &val);
+	ret = kstrtoul(buf, 10, &val);
 	if (ret < 0)
 	{
 		return ret;
@@ -989,7 +989,7 @@ static ssize_t user_settings_write(struct file *filp, const char __user *ubuf, s
 	}
 	buf[cnt] = '\0';
 
-	ret = strict_strtoul(buf, 10, &val);
+	ret = kstrtoul(buf, 10, &val);
 	if (0 != ret)
 	{
 		return ret;
