diff --git a/driver/src/devicedrv/mali/linux/mali_osk_time.c b/driver/src/devicedrv/mali/linux/mali_osk_time.c
index b399b87..f3fa478 100644
--- a/driver/src/devicedrv/mali/linux/mali_osk_time.c
+++ b/driver/src/devicedrv/mali/linux/mali_osk_time.c
@@ -14,8 +14,7 @@
  */
 
 #include "mali_osk.h"
-#include <linux/jiffies.h>
-#include <linux/time.h>
+#include <linux/ktime.h>
 #include <asm/delay.h>
 
 int	_mali_osk_time_after( u32 ticka, u32 tickb )
