commit 80c6e6f4bb5b0a034c6e291a77c87ef653a16ea1
Author: Robbert-Jan de Jager <robojan1@hotmail.com>
Date:   Thu Oct 15 16:37:05 2015 +0200

    Fixed some bugs, Note: Dirty fix of onegit status!

diff --git a/drivers/video_sunxi/sunxi_v3/de/lowlevel_sun8iw7/de_wb.c b/u-boot-2011.09/drivers/video_sunxi/sunxi_v3/de/lowlevel_sun8iw7/de_wb.c
index 3a1bb1b..607cc77 100644
--- a/drivers/video_sunxi/sunxi_v3/de/lowlevel_sun8iw7/de_wb.c
+++ b/drivers/video_sunxi/sunxi_v3/de/lowlevel_sun8iw7/de_wb.c
@@ -272,6 +272,7 @@ __s32 WB_EBIOS_Set_Para(__u32 sel, struct disp_capture_config *cfg)
 		wb_dev[sel]->bypass.dwval |= 0x00000001;
 	}
 	//Coarse scaling
+	cs_out_w0 = cs_out_h0 = 0; // TODO not checked!!!
 	if((crop_w > (out_window_w<<1))&&(crop_h > (out_window_h<<1)))
 	{
 		wb_dev[sel]->bypass.dwval |= 0x00000002;
diff --git a/mkconfig b/u-boot-2011.09/mkconfig
old mode 100644
new mode 100755
diff --git a/tools/scripts/README b/u-boot-2011.09/tools/scripts/README
old mode 100644
new mode 100755
diff --git a/tools/scripts/define2mk.sed b/u-boot-2011.09/tools/scripts/define2mk.sed
old mode 100644
new mode 100755
diff --git a/tools/scripts/dot.kermrc b/u-boot-2011.09/tools/scripts/dot.kermrc
old mode 100644
new mode 100755
diff --git a/tools/scripts/flash_param b/u-boot-2011.09/tools/scripts/flash_param
old mode 100644
new mode 100755
diff --git a/tools/scripts/make-asm-offsets b/u-boot-2011.09/tools/scripts/make-asm-offsets
old mode 100644
new mode 100755
diff --git a/tools/scripts/send_cmd b/u-boot-2011.09/tools/scripts/send_cmd
old mode 100644
new mode 100755
diff --git a/tools/scripts/send_image b/u-boot-2011.09/tools/scripts/send_image
old mode 100644
new mode 100755
diff --git a/usb_sunxi/usb_burn.c b/u-boot-2011.09/usb_sunxi/usb_burn.c
index b9d0dfa..afcc6da 100644
--- a/usb_sunxi/usb_burn.c
+++ b/usb_sunxi/usb_burn.c
@@ -31,8 +31,8 @@
 #include <smc.h>
 #include <securestorage.h>
 #include "key_deal.h"
-
-int smc_set_sst_crypt_name(char *name);
+
+int smc_set_sst_crypt_name(char *name);
 
 static  int sunxi_usb_pburn_write_enable = 0;
 #if defined(SUNXI_USB_30)
@@ -458,11 +458,11 @@ int __sunxi_burn_key(u8 *buff, uint buff_len)
 		printf("key len=%d\n", key_list->len);
 		printf("key if_burn=%d\n", key_list->if_burn);
 		printf("key if_replace=%d\n", key_list->if_replace);
-		printf("key if_crypt=%d\n", key_list->if_crypt);
+		printf("key if_crypt=%d\n", key_list->if_crypt);
 		printf("key data:\n");
 		sunxi_dump(key_list->key_data, key_list->len);
 		printf("###################\n");
-		offset = (sizeof(sunxi_usb_burn_key_info_t)) + ((key_list->len + 15) & (~15));
+		offset = (sizeof(sunxi_usb_burn_key_info_t)) + ((key_list->len + 15) & (~15));
 		printf("offset=%d\n", offset);
 		p_buff += offset;
 #ifdef CONFIG_SUNXI_SECURE_SYSTEM
@@ -497,10 +497,10 @@ int __sunxi_burn_key(u8 *buff, uint buff_len)
 			else
 #endif
 			{
-			if(key_list->if_crypt)
-				smc_set_sst_crypt_name(key_list->name);
-
-				if(sunxi_secure_object_write(key_list->name, (char *)key_list->key_data, key_list->len))
+			if(key_list->if_crypt)
+				smc_set_sst_crypt_name(key_list->name);
+
+				if(sunxi_secure_object_write(key_list->name, (char *)key_list->key_data, key_list->len))
 				{
 					return -1;
 				}
@@ -1075,7 +1075,8 @@ static int sunxi_pburn_state_loop(void  *buffer)
 						case 1:				//小机端接收数据
 						{
 							//pburn_flash_sectors  = *(int *)(cbw->CBWCDB + 8);
-							pburn_flash_start    = *(int *)(cbw->CBWCDB + 4);
+							//pburn_flash_start    = *(int *)(cbw->CBWCDB + 4);
+							memcpy(&pburn_flash_start, cbw->CBWCDB + 4, sizeof(int));
 
 							trans_data.recv_size = cbw->dCBWDataTransferLength;
 							trans_data.act_recv_buffer = (uint)trans_data.base_recv_buffer;
@@ -1092,8 +1093,10 @@ static int sunxi_pburn_state_loop(void  *buffer)
 						{
 							uint start, sectors;
 
-							start   = *(int *)(cbw->CBWCDB + 4);		//读数据的偏移量
-							sectors = *(int *)(cbw->CBWCDB + 8);		//扇区数;
+							//start   = *(int *)(cbw->CBWCDB + 4);		//读数据的偏移量
+							//sectors = *(int *)(cbw->CBWCDB + 8);		//扇区数;
+							memcpy(&start, cbw->CBWCDB + 4, sizeof(int));
+							memcpy(&sectors, cbw->CBWCDB + 8, sizeof(int));
 
 							printf("start=%d, sectors=%d\n", start, sectors);
 
