diff --git a/u-boot-2011.09/arch/arm/cpu/armv7/config.mk b/u-boot-2011.09/arch/arm/cpu/armv7/config.mk
index 45c6acb..362aeed 100644
--- a/arch/arm/cpu/armv7/config.mk
+++ b/arch/arm/cpu/armv7/config.mk
@@ -21,9 +21,9 @@
 # MA 02111-1307 USA
 #
 ifndef CONFIG_SPL
-PLATFORM_RELFLAGS += -fno-common -ffixed-r8 -msoft-float
+PLATFORM_RELFLAGS += -fno-common -ffixed-r8 -mfloat-abi=hard -mfpu=neon-vfpv4
 else
-PLATFORM_RELFLAGS += -fno-common -msoft-float
+PLATFORM_RELFLAGS += -fno-common -mfloat-abi=hard -mfpu=neon-vfpv4
 endif
 # Make ARMv5 to allow more compilers to work, even though its v7a.
 ifdef CONFIG_ARM_A8
