diff --git a/driver/src/devicedrv/ump/Kbuild b/driver/src/devicedrv/ump/Kbuild
index 042745d..407bf96 100644
--- a/driver/src/devicedrv/ump/Kbuild
+++ b/driver/src/devicedrv/ump/Kbuild
@@ -45,6 +45,8 @@ else
 ccflags-y += -I$(src)/linux/license/gpl
 endif
 
+ccflags-y += -Wno-date-time
+
 ump-y = common/ump_kernel_common.o \
 	common/ump_kernel_descriptor_mapping.o \
 	common/ump_kernel_api.o \
diff --git a/driver/src/devicedrv/ump/linux/ump_osk_low_level_mem.c b/driver/src/devicedrv/ump/linux/ump_osk_low_level_mem.c
index 0073c4d..16c2c29 100644
--- a/driver/src/devicedrv/ump/linux/ump_osk_low_level_mem.c
+++ b/driver/src/devicedrv/ump/linux/ump_osk_low_level_mem.c
@@ -148,7 +148,7 @@ _mali_osk_errcode_t _ump_osk_mem_mapregion_init( ump_memory_allocation * descrip
 
 	vma->vm_private_data = vma_usage_tracker;
 	vma->vm_flags |= VM_IO;
-	vma->vm_flags |= VM_RESERVED;
+	vma->vm_flags |= (VM_DONTEXPAND | VM_DONTDUMP);
 
 	if (0==descriptor->is_cached)
 	{
